### DISCUSSION TITLE
How to repair or recreate?

### DISCUSSION THREAD

**Discussion by ghost** [borgbase/vorta#1545] [FAQ]
Hi,

I just got the following error:
![Screenshot 2023-01-17 at 09 24 35](https://user-images.githubusercontent.com/34644672/212860635-9d264e8e-eaf6-4be4-882c-df9e77d5596a.png)

I cannot find any option to do either of these things.  How does one repair a repository?  Does recreate mean start again from scratch?

Thanks!

**Comment by m3nu** [comment]
Hi, this can't be done inside Vorta, since it's relatively rare and you should look at the console output.

That said, this may be harmless and could simple be an interrupted backup or a known issue that a few users reported after using Borg 1.2.x.

What you'll want to do is run `borg check ssh://` and then `borg check --repair ssh://` on the repo. More here: https://docs.borgbase.com/faq/#repair-damaged-repositories

**Reply by ghost** [reply]
Thanks.

Running `borg check ssh://` in terminal on my Mac results in `command not found: borg`
